examples/path-line.js

```diff
@@ -39,6 +37,6 @@ let svg = `<svg xmlns="http://www.w3.org/2000/svg">
     <line x1="0" y1="-10" x2="110" y2="110" stroke="green"/>
     ${intersectionSVG}
   </g>
-</svg>`
+</svg>`;
```

Comment:
Admittedly, I didn't review the examples as thoroughly as I should have. Seeing this semicolon made me realize there is one missing in `example-2.js` at the end of the same statement, fwiw

Reply:
Got it, sure, I plan to apply shortly. Btw, if you want stricter rules than just `eslint:recommended` (a config built in to ESLint), I have my own [`eslint-config-ash-nazg`](https://github.com/brettz9/eslint-config-ash-nazg) config for ESLint which are derived from a number of plugins/confis and are quite strict (though not ridiculously strict) and good at catching stylistic inconsistencies as well.

Reply:
I'm sure there are all sorts of stylistic issues in this code :). I'm open to improved settings, but I'd be curious to see what changes it would cause, of course

Reply:
I've now applied.

Reply:
Sure. I can also inherit the config but explicitly override all of the specific rules (in the main `.eslintrc.js` config file) which add any significant stylistic changes, allowing you to reenable if you wish. Most of my config's stylistic rules are based on "standard", a config intended to standardize some stylistic conventions, though I've overridden a few in my own config which didn't seem common practice in the JS community.